RegenerateServiceKey/index.ts

```diff
@@ -0,0 +1,40 @@
+import { Context } from "@azure/functions";
+import * as express from "express";
+import { cosmosdbInstance } from "../utils/cosmosdb";
+
+import {
+  SERVICE_COLLECTION_NAME,
+  ServiceModel
+} from "io-functions-commons/dist/src/models/service";
+import { secureExpressApp } from "io-functions-commons/dist/src/utils/express";
+import { setAppContext } from "io-functions-commons/dist/src/utils/middlewares/context_middleware";
+
+import createAzureFunctionHandler from "io-functions-express/dist/src/createAzureFunctionsHandler";
+
+import { getApiClient } from "../utils/apiclient";
+import { GetRegenerateServiceKey } from "./handler";
+
+// Setup Express
+const app = express();
+secureExpressApp(app);
+
+const client = getApiClient();
+
+const serviceModel = new ServiceModel(
+  cosmosdbInstance.container(SERVICE_COLLECTION_NAME)
+);
+
+app.get(

```

Comment:
```suggestion
app.put(

```

Reply:
Fixed in [98db5c7](https://github.com/pagopa/io-functions-services/pull/68/commits/98db5c77eefe2f2bbce99559596e00feed83aa0c)